src/packages/cli/package.json

```diff
@@ -118,20 +116,17 @@
     "tempy": "0.6.0",
     "terminal-link": "2.1.1",
     "ts-jest": "26.3.0",
-    "ts-mocha": "7.0.0",
     "typescript": "4.0.2",
     "ws": "7.3.1"
   },
   "scripts": {
-    "test:sqlite": "jest integrate.sqlite",
-    "test:mariadb": "./node_modules/.bin/mocha src/__tests__/integrate.mariadb.test.ts --require ts-mocha src/__tests__/integrate.mariadb.test.ts --timeout 10s",
-    "test:mysql": "./node_modules/.bin/mocha src/__tests__/integrate.mysql.test.ts --require ts-mocha src/__tests__/integrate.mysql.test.ts --timeout 10s",
-    "test:postgresql": "./node_modules/.bin/mocha src/__tests__/integrate.postgresql.test.ts --require ts-mocha src/__tests__/integrate.postgresql.test.ts --timeout 10s",
+    "test:sqlite": "jest integration.sqlite --forceExit",
+    "test:postgresql": "jest integration.postgresql --forceExit",
+    "test:mysql": "jest integration.mysql --forceExit",
+    "test:mariadb": "jest integration.mariadb --forceExit",
     "test:commands": "./fixtures/test.sh && jest dotenv- format doctor generate studio && jest introspect --forceExit",
-    "test": "pnpm run test:commands && pnpm run test:postgresql && jest integrate.sqlite && pnpm run test:mysql",
-    "test:debug": "./node_modules/.bin/mocha --inspect-brk src/__tests__/integrate.* --require ts-mocha src/__tests__/integrate.* --timeout 10s",
-    "test:mocha": "pnpm run test:sqlite && pnpm run test:postgresql && pnpm run test:mysql",
```

Comment:
Do we use this much? Should I try to find a version of this that works for Jest?

Reply:
I don't understand why you removed the main "test" 🤔 

It's ok to get rid of debug though

Because how does `pnpm run test` work then?

Reply:
`"test"` is still there :) 

Just a not-great diff rendering

Reply:
Oh ok 😓